packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -314,6 +314,10 @@ abstract class Generator {
   /// Write the generated code described in [root] to [sink] using the
   /// [options].
   void generate(StringSink sink, PigeonOptions options, Root root);
+
+  /// Generates errors that would only be appropriate for the given generator.
```

Comment:
"given generator" sounds like it's a param; how about "this generator". Or for consistency with the `shouldGenerate` comment, "the [Generator]". (It seems odd to use a class reference to describe an *instance* of the class though.)

Reply:
done